src/Components/ActivationKey/AdditionalRepositoriesCard.js

```diff
@@ -34,45 +36,48 @@ const AdditionalRepositoriesCard = (props) => {
       !isEditAdditionalRepositoriesModalOpen
     );
   };
-
-  const ButtonWrapper = () => {
+  const AdditionalReposCard = () => {
     return (
-      <AddAdditionalRepositoriesButton
-        onClick={handleEditAdditionalRepositoriesToggle}
-      />
+      <Card>
+        <CardHeader>
+          <CardTitle>
+            <Title headingLevel="h2"> Additional repositories </Title>
+          </CardTitle>
+        </CardHeader>
+        <CardBody>
+          <TextContent>
+            <Text component={TextVariants.p}>
+              The core repositories for your operating system version, for
+              example BaseOS and AppStream, are always enabled and do not need
+              to be explicitly added to the activation key.
+            </Text>
+            <AddAdditionalRepositoriesButton
+              onClick={handleEditAdditionalRepositoriesToggle}
+            />
+            <AddAdditionalRepositoriesModal
+              title="Additional Repositories"
+              isOpen={isEditAdditionalRepositoriesModalOpen}
+              handleModalToggle={handleEditAdditionalRepositoriesToggle}
+              modalSize="large"
+              repositories={availableRepositories}
+              isLoading={isLoading}
+              error={error}
```

Comment:
`title` and `modalSize` are no longer passed to this component. See #176 where that was removed.